### PR TITLE
Fix for selected index when switching scale

### PR DESCRIPTION
**Pull request by pavelkeyzik**
This PR fixes the issue #19.

I've updated the code to check if our selected index is greater than our color scale, and if it is, then we just select the last color from the scale, but if we switch back, it will be the same selected color.

@rafaeltab Could you test it and let me know if everything works as expected? 😊

### CODE REVIEW

**Comment by pavelkeyzik** [comment]
Oh... Sorry! I've just realized that to test it we need the access 😄 Maybe Cole Bemis will came up with something that will allow us to test it.

File: src/pages/scale.tsx
Comment (stevenpetryk) [inline src/pages/scale.tsx:53]: This could be a bit more succinct with Math.min:

```ts
const index = Math.min(+selectedIndex, scale.colors.length - 1).toString()
```

Comment (pavelkeyzik) [reply src/pages/scale.tsx]: Fixed 😊

File: src/pages/scale.tsx
Comment (colebemis) [inline src/pages/scale.tsx:50]: [Minor] I'd prefer to be a bit more explicit when casting `selectedIndex` to a number:

```suggestion
  const index = String(Math.min(Number(selectedIndex), scale.colors.length - 1));
```

**Review by colebemis — APPROVED** [review]
👍 Thanks for fixing this, @pavelkeyzik!

**Comment by colebemis** [comment]
> Oh... Sorry! I've just realized that to test it we need the access 😄 Maybe Cole Bemis will came up with something that will allow us to test it.

We actually switched from Netlify to GitHub Pages for deploys (still need to remove the Netlify integration 😅). I'll chat with @rezrah about how we can let contributors deploy previews of the app.

**Comment by colebemis** [comment]
@pavelkeyzik I'm happy to merge after the merge conflicts are resolved 👍

File: src/pages/scale.tsx
Comment (pavelkeyzik) [reply src/pages/scale.tsx]: I absolutely agree with you 👍 Fixed that as well 😊

**Comment by pavelkeyzik** [comment]
@colebemis Thanks for review 😊 I've fixed merge conflicts

**Pull request merged**
